### PR TITLE
Create Express Server

### PR DESCRIPTION
**Pull request by gato-n0ir**
create express server to listen for :stockTicker /'query' 
fetch 'query' tables from SQL database 
EDIT: Added - create {x/y} coordinates for graph

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    //build new Graph from data
+
+var db = mysql.createConnection({
+    host: "localhost",
+    user: "root",
+    password: "password",
+    database: "tradeDeskdb"
+});
+
+// app.get('/', (req, res) => res.send('Hello World!'))
```
Comment (jaytxng) [inline server/server.js:25]: reminder to take out the stray code before making the PR

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    coordinates.x = i
+                    coordinates.y = results[i].price
+                dataPoints.push(coordinates)
+        }
+        res.send(dataPoints)  //creates x and y points out of i and price for graphing
+    });
+
+});
+
+app.listen(port, () => console.log(`Example app listening on port ${port}!`))
```
Comment (jaytxng) [inline server/server.js:45]: Be sure to update the messaging when app is running

**Review by jaytxng — APPROVED** [review]
Looks like db/seederFunction.js and db/seed.js were already included in another PR. Would advise keeping these commits in their own PRs, even though we're not expecting any conflicts.

File: server/server.js (hunk 1 above)
Comment (gato-n0ir) [reply server/server.js:25]: Thank you! Will make changes.

File: server/server.js (hunk 2 above)
Comment (gato-n0ir) [reply server/server.js:45]: 💯

**Pull request merged**